jupyterlab-server-proxy/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "@jupyterlab/server-proxy",
-  "version": "2.1.2",
-  "description": "Launcher icons for proxied applications",
+  "version": "3.0.0",
```

```suggestion
  "version": "3.0.0-rc.1",
```
@consideRatio shall we go for an rc instead? If so I'll open a PR to automate the publish to pypi and npm steps after this is merged. I've got an example in https://github.com/manics/jupyter-offlinenotebook/blob/5846927739fd33c35a2950874ce904c9b113ef33/.github/workflows/build.yml#L118-L161

---

@manics either is fine in my mind!

I'm not fully confident on the practices of pre-release formatting in python packages btw. Is it okay to name the pre-release `-rc.1` or "should it" be something like `rc1`?

---

npm versioning is stricter than pypi, so using `rc.1` as required by npm will work with pypi. See for example
- https://github.com/manics/jupyter-offlinenotebook/commit/28e7072bbe23639e76d7fa08460e3564cca9622c
- https://pypi.org/project/jupyter-offlinenotebook/0.2.0rc1/
- https://www.npmjs.com/package/jupyter-offlinenotebook/v/0.2.0-rc.1

---

The reason I suggested an rc if we're going with an automated publish workflow is that you can't delete a release from pypi ot npm, and since we'll be doing a synchronised release of both packages it'd be safer to test it first.

---

Aha!!! PyPi strips `-` and `.` if we suffix `-rc.1`, that explains a lot. Then absolutely go for `-rc.1`!